CMakeLists.txt

```
@@ -42,6 +42,7 @@ option(ENABLE_BACKTRACE "Build Umpire with allocation backtrace enabled" Off)
 option(ENABLE_BACKTRACE_SYMBOLS "Build Umpire with symbol support" Off)
 option(ENABLE_COVERAGE "Enable code coverage (with GCC)" Off)
 option(ENABLE_PEDANTIC_WARNINGS "Enable pedantic compiler warnings" On)
+option(ENABLE_ACCESS_CHECK "Enable testing for allocator inaccessibility" On)
```

To me, the name makes it seem like is it a flag to simply enable/disable accessibility checking.  Perhaps this could be more clearly named something like:
```suggestion
option(ENABLE_ACCESSILITY_TESTS "Test allocator inaccessibility functionality" On)
```

---

Yea, I've been pondering whether or not that was really the best name for it. But with your suggestion, what do you think of "ENABLE_INACCESSIBILITY_TESTS"?

---

Done. Thanks :)